keras/optimizers/optimizer_experimental/adamw.py

```diff
@@ -215,6 +221,15 @@ def get_config(self):
     })
     return config
 
+  def exclude_from_weight_decay(self, var_list):
+    if self._built:
+      if var_list == None:
+        self._exclude_from_weight_decay = []
+      else:
+        self._exclude_from_weight_decay = var_list
+    else:
+      raise ValueError('This optimizer has not yet been built. '
```

Comment:
The check should work the other way around: the user need to call `exclude_from_weight_decay` before the optimizer has been built.

Reply:
Sorry.